src/main/java/com/uid2/shared/encryption/AesGcm.java

```diff
@@ -32,6 +33,23 @@ public static EncryptedPayload encrypt(String s, EncryptionKey key) {
         }
     }
 
+    public static EncryptedPayload encrypt(byte[] b, KeysetKey key) {
+        try {
+            byte[] encrypted = encrypt(b, key.getKeyBytes());
```

Comment:
Can we make common functions or remove the old functions?

Reply:
Did you test that the old functions are safe to remove? This is technically a breaking changed to shared, so we should do a major release